<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Laptop" name="tests.TC001findby" tests="1" failures="0" timestamp="19 Oct 2017 14:21:13 GMT" time="46.469" errors="1">
  <testcase name="FindBy" time="46.469" classname="tests.TC001findby">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at wrappers.GenericWrappers.enterById(GenericWrappers.java:48)
	at pages.LoginPage.typeUsername(LoginPage.java:25)
	at tests.TC001findby.FindBy(TC001findby.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- FindBy -->
</testsuite> <!-- tests.TC001findby -->
